script/bulk_import/base.rb

```diff
@@ -484,7 +486,8 @@ def create_records(rows, name, columns)
         mapped = yield(row)
         next unless mapped
         processed = send(process_method_name, mapped)
-        imported_ids << mapped[:imported_id]
+        imported_ids << mapped[:imported_id] unless mapped[:imported_id].nil?
+        imported_ids |= mapped[:imported_ids] unless mapped[:imported_ids].nil?
```

Comment:
This is for derived classes.

I am currently extends `BulkImport::VBulletin` for my case and think that imported_ids may be useful for future works.

I combined some old categories into new categories. I am having .yml file that have a list of new categories and imported_ids which are ids of old categories.